server/src/routes/donatedItemRoutes.ts

```diff
@@ -55,17 +56,64 @@ router.post('/', donatedItemValidator, async (req: Request, res: Response) => {
 // GET /donatedItem - Fetch all donated items
 router.get('/', async (req: Request, res: Response) => {
     try {
-        const items = await prisma.donatedItem.findMany({
+    const donatedItem = await prisma.donatedItem.findMany({
```

Comment:
`findMany `returns list of records. So `items` or `donatedItems` variable name looks meaningful, please change the variable name accordingly

Reply:
Great! I will update accordingly